### PR TITLE
[record-minmax] Tidy RecordMinMax

### PR DESCRIPTION
**Pull request by jinevening**
This tidies RecordMinMax source code.
- Remove unused code
- Use proper test name

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/14255
Draft PR: https://github.com/Samsung/ONE/pull/14257

### CODE REVIEW

File: compiler/record-minmax/CMakeLists.txt [hunk 1]
```diff
@@ -64,7 +64,7 @@ set(TEST_SOURCES
 file(GLOB_RECURSE TESTS "tests/*.test.cpp")
 
 nnas_find_package(GTest REQUIRED)
-GTest_AddTest(record_minmax_function_test ${TESTS} ${TEST_SOURCES})
-target_include_directories(record_minmax_function_test PRIVATE include)
-target_link_libraries(record_minmax_function_test luci_lang)
-target_link_libraries(record_minmax_function_test nncc_coverage)
+GTest_AddTest(record_minmax_unittest ${TESTS} ${TEST_SOURCES})
```
Comment (jinevening) [inline compiler/record-minmax/CMakeLists.txt:67]: I renamed this test because unittest is a more widely-used term.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by Seunghui98 — APPROVED** [review]
LGTM :)

**Review by SlavikMIPT — APPROVED** [review]
LGTM

**Pull request merged**